xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2240,7 +2240,7 @@ admin.colortheme.manage=Manage color themes\u00BB
 admin.customize=Customize
 admin.save=Save
 admin.defaultwikinotinstalled=Your wiki seems empty. You may want to import the default XWiki Enterprise wiki which contains a set of useful pages: user profiles, recent activity, administration pages and many more. This wiki is distributed as a XAR file, you can download it from {0}.
-admin.adminappnotinstalled=The administration application is not installed. Since XWiki Enterprise 1.5 the Administration is distributed as an application. You can download it from {0}.
+admin.adminappnotinstalled=The administration application is not installed. You should install it with the {0}Extension Manager{1}.
```

Comment:
The comment made by @vmassol [here](https://github.com/xwiki/xwiki-platform/pull/680#discussion_r160031901) should not apply. We can't change the parameters of a translation key that easily.